client/dist/style.css

```diff
@@ -256,13 +256,8 @@ align-items: center;
 
 .carouselCard {
   display: inline-block;
-  height: 500px;
-  width: 200px;
-  margin: 16px;
-}
-
-.carouselCard p {
-  margin: 0px;
+  height: 300px;
```

Comment:
Maybe use rem's here instead of pixels for the sake of responsiveness/accessibility. Generally can get a good estimate of how many rems you need by dividing your current number of pixels by 16, this is the default font size for most browsers. In this case, 300px/16 giving you 18.75rem.